.travis.yml

```diff
@@ -34,14 +34,7 @@ install:
       conda install --yes --quiet conda-forge-build-setup
       source run_conda_forge_build_setup
 
-      # install conda-build 2.x to build with a long prefix
-      conda install --yes --quiet conda-build=2
-      conda info
```

Comment:
Do we really want to merge this? I guess that is about time we start using conda-build 2 already.
Maybe we should  even add the `inspect prefix` to `conda-smithy` in order to  fail on short prefixes, so we can and fix them. @conda-forge/core